src/core/control/Control.h

```diff
@@ -88,6 +88,7 @@ class Control:
     // Menu File
     bool newFile(std::string pageTemplate = "", fs::path filepath = {});
     bool openFile(fs::path filepath = "", int scrollToPage = -1, bool forceOpen = false);
+    bool changePdfBackground(fs::path filepath = "", int scrollToPage = -1, bool forceOpen = false);
```

Comment:
The parameters `filepath`, `scrollToPage` and `forceOpen` are not used anywhere. I think they should be removed.